<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sakti_C" failures="1" tests="1" name="runners.RerunRunner" time="9.968" errors="0" timestamp="2025-04-14T20:18:13 IST" skipped="0">
  <testcase classname="runners.RerunRunner" name="runScenario" time="9.968">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepDefs.BookSearchStepDef.should_display_crime_result_page(BookSearchStepDef.java:53)
at âœ½.Should display crime result page(file:///E:/eclipse/workspace/BDDSamples/src/test/resources/features/Booksearch.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runners.RerunRunner -->
